css/style.css

```
@@ -140,6 +196,22 @@ header .toggle {
 
 }
 
+@media only screen and (max-width:767px) {
+  
+  .feature .inner .wrapper {
+    flex-wrap:wrap;
```

ここは折り返すのではなく、縦列に変える方が相応しい気がします。
```suggestion
    flex-direction: column;
```

---

> ここは折り返すのではなく、縦列に変える方が相応しい気がします。

修正しました。
bd7c4650b7fe7c3dbc6f44d265380cbe96faefea